tests/unit/utils/test_validators.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+from kolena._utils.validators import validate_not_blank
+
+
+def test__validate__field_name__empty() -> None:
+    with pytest.raises(ValueError):
+        validate_not_blank("")
+
+    with pytest.raises(ValueError):
+        validate_not_blank("  ")
```

Comment:
Minor, but I'd use a parametrize fixture here and test more whitespace variants

Reply:
Updated